recipes/default.rb

```diff
@@ -38,6 +38,7 @@
 end
 
 runit_service "irccat" do
+    default_logger true
```

Comment:
This needs a `log_dir ./main` for backwards compatibility.

If you like, add a `default[:irccat][:log_dir]` attribute to allow this to be customised. With a note that a value of `nil` defaults the directory to `/var/log/irccat/`